.eslintrc

```diff
@@ -8,7 +8,8 @@
     "eslint-config-uber-es2015"
   ],
   "rules": {
-    "guard-for-in": 0
+    "guard-for-in": 0,
+    "callback-return": 0
```

Comment:
Do we need to disable `callback-return` here in the root? This is related to node.js style callbacks which we don't use a lot.

Seems related to the demo, could we put it in a comment in the offending file or in a local .eslint for the demo?

Just a nit, the motivation being to keep our top level eslintrc small.